README.md

```diff
@@ -1,21 +1,51 @@
-* `iris_simple_example.ipynb` - tutorial showing how to use `cleanlab` on the simple IRIS dataset.
-* `model_selection_demo.ipynb` - tutorial showing model selection on the cleanlab's parameter settings.
-* `simplifying_confident_learning_tutorial.ipynb` - tutorial implementing cleanlab as raw numpy code.
-* `visualizing_confident_learning.ipynb` - tutorial to demonstrate the noise matrix estimation performed by cleanlab.
+Recommended order of examples to try:
+
+| Notebook                                                                                                                                        | Description                                                                                                            |
+| ----------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| [iris_simple_example.ipynb](https://github.com/cleanlab/examples/blob/master/iris_simple_example.ipynb)                                         | Use `cleanlab` to find synthetic label errors in the Iris dataset.                                                     |
+| [classifier_comparison.ipynb](https://github.com/cleanlab/examples/blob/master/classifier_comparison.ipynb)                                     | Demonstrate how `cleanlab` can be used to train 10 different classifiers on 4 dataset distributions with label errors. |
+| [model_selection_demo.ipynb](https://github.com/cleanlab/examples/blob/master/model_selection_demo.ipynb)                                       | Perform hyperparameter optimization with `cleanlab`'s hyperparameters.                                                 |
+| [simplifying_confident_learning_tutorial.ipynb](https://github.com/cleanlab/examples/blob/master/simplifying_confident_learning_tutorial.ipynb) | Implement `cleanlab` as raw numpy code.                                                                                |
+| [visualizing_confident_learning.ipynb](https://github.com/cleanlab/examples/blob/master/visualizing_confident_learning.ipynb)                   | Demonstrate how `cleanlab` performs noise matrix estimation.                                                           |
+
+## Instructions
+
+To run the latest example notebooks, execute the commands below which will install the required libraries in a virtual environment.
+
+```console
+python -m pip install virtualenv
+python -m venv env
+source env/bin/activate
+python -m pip install -r requirements.txt
+```
+
+Run the notebooks individually or run either scripts below which will execute and save each notebook.
+
+Bash script:
+
+```console
+./run_all_notebooks.sh
+```
+
+Python script:
+
+```console
+python run_all_notebooks.py
+```
+
+## Old Examples
```

Comment:
[Nit] Old -> Older

Reply:
Good catch, agree this should be "Older". I made this change. Thanks.